products/mariadb.md

```diff
@@ -103,10 +103,10 @@ releases:
 
 ---
 
-> [MariaDB](https://mariadb.org/about/) is a community-developed, commercially supported fork of the MySQL relational database management system (RDBMS).
+> [MariaDB](https://mariadb.org/about/) is a community-developed, commercially supported relational database management system (RDBMS) originally forked from MySQL.
 
-A new MariaDB Server version will be announced for General Availability approximately once a year. The MariaDB Foundation guarantees that every release will be maintained for at least 5 years.
+A new MariaDB Server version will be announced one per quarter bringing new features to the server. These release series are maintainced for approximately once a year after their General Availability (stable) date.
 
-The MariaDB Foundation may decide to maintain selected releases beyond the normal 5 years if they are in exceptionally widespread use. As MariaDB 5.5 is included in multiple prominent Linux distributions with a lifespan beyond 2017, the Foundation is committed in maintaining it until 2020.
+There will be long term support releases selected that will be maintained for five years. This long term support series will be selected approximately every 2 years.
```

Comment:
```suggestion
Some release series are selected for longer maintenance windows approximately every 2 years. These long-term support release series are maintained for five years with security and major bug fixes.
```

Rewrote the sentences with the term "release series" used in the previous paragraph. Also inverted the two sentence (it feels more natural to me) and clarify a little bit what LTS means (information found on https://mariadb.org/about/#maintenance-policy).

Reply:
technically "since 10.6".  and "with security and bug fixes" - there's no criteria on bug size. Ok with rest.

Reply:
Ok, I have updated the suggestion and applied it.